src/content/app/genome-browser/components/zmenu/Zmenu.tsx

```diff
@@ -51,15 +58,66 @@ const Zmenu = (props: ZmenuProps) => {
 
   const destroyZmenu = () => {
     dispatch(changeHighlightedTrackId(''));
-    setZmenus && setZmenus(pickBy(zmenus, (value, key) => key !== props.id));
+    setZmenus &&
+      setZmenus(pickBy(zmenus, (value, key) => key !== props.zmenuId));
   };
 
   const direction = chooseDirection(props);
   const toolboxPosition =
     direction === Direction.LEFT ? ToolboxPosition.LEFT : ToolboxPosition.RIGHT;
 
+  const { content, variety } = props.payload;
+  const features: (ZmenuContentTranscript | ZmenuContentGene)[] = [];
+
+  // get the first variety type
```

Comment:
this comment is probably no longer necessary?

Reply:
Removed it